tests/layer_tests/mo_python_api_tests/test_mo_convert_paddle.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+from common.mo_convert_test_class import CommonMOConvertTest
+
+import openvino
+import openvino.runtime as ov
+from openvino.runtime import PartialShape, Model, Dimension, serialize
```

Comment:
please cleanup imports. In PyCharm just use "Ctrl+A", "Ctrl+Alt+O" and "Ctrl+Alt+L"

Reply:
> please cleanup imports. In PyCharm just use "Ctrl+A", "Ctrl+Alt+O" and "Ctrl+Alt+L"

Thanks. cleaned them.